app/Http/Controllers/api/v1/auth/RegisterController.php

```diff
@@ -62,4 +66,40 @@ protected function create(array $data)
             'password' => Hash::make($data['password']),
         ]);
     }
+
+    /**
+     * Create a new user through invitation register method
+     * @param  StoreUserInvitation $request
+     * @return JsonResponse
+     */
+    public function invitationRegister(StoreUserInvitation $request)
+    {
+        $user = new User();
+
+        // Find the invitation record based on requests
+        $invitation = Invitation::query()
+            ->where('invitation_token', $request->invitation_token)
+            ->where('email', $request->email)
```

Comment:
I wouldn't check the email, as it could be that the user wants to use a different email for signup

Reply:
https://github.com/THM-Health/PILOS/pull/23/commits/1a33a7aa89dddde61afd7c0994421ca8d1e20bfb